homeassistant/components/flexmeasures/config_flow.py

```diff
@@ -0,0 +1,239 @@
+"""Config flow for FlexMeasures integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from flexmeasures_client import FlexMeasuresClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry, OptionsFlow
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(
+            "url", description={"suggested_value": "http://localhost:5000"}
+        ): str,
+        vol.Required(
+            "username", description={"suggested_value": "toy-user@flexmeasures.io"}
+        ): str,
+        vol.Required("password"): str,
+        vol.Required("power_sensor", description={"suggested_value": 1}): int,
+        vol.Required(
+            "consumption_price_sensor", description={"suggested_value": 2}
+        ): int,
+        vol.Required(
+            "production_price_sensor", description={"suggested_value": 2}
+        ): int,
+        vol.Required("soc_sensor", description={"suggested_value": 4}): int,
+        vol.Required("rm_discharge_sensor", description={"suggested_value": 5}): int,
+        vol.Required(
+            "schedule_duration", description={"suggested_value": "PT24H"}
+        ): str,
```

Comment:
Please add `data_description` in strings.json to explain to the user how to fill in this form